flusher.go

```diff
@@ -126,6 +129,28 @@ func (s *Server) Flush(ctx context.Context) {
 	}()
 }
 
+type timeseriesSummary struct {
+	tags  []string
+	count int64
+}
+
+func (s *Server) tallyTimeseries() timeseriesSummary {
+	tags := []string{fmt.Sprintf("global_veneur:%t", !s.IsLocal())}
+	count := int64(0)
+
+	for _, w := range s.Workers {
+		w.totalMTSMtx.Lock()
+		count += int64(w.totalMTS.Hll.Estimate())
```

Comment:
Yikes, this is actually a bug. We want to merge the sets together, not `+=` their HLL estimates.